openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataQualityAndProfiler.spec.js

```diff
@@ -465,16 +465,109 @@ describe('Data Quality and Profiler should work properly', () => {
       .should('eq', 'No data');
   });
 
-  it('Delete Test suite should work properly', () => {
+  it('Soft Delete Test suite should work properly', () => {
     cy.get('[data-testid="appbar-item-data-quality"]')
       .should('be.visible')
       .click();
     cy.get(`[data-row-key="${NEW_TEST_SUITE.name}"] > :nth-child(1) > a`)
       .contains(NEW_TEST_SUITE.name)
       .should('be.visible')
       .click();
-    cy.get('[data-testid="test-suite-delete"]').should('be.visible').click();
-    cy.get('[data-testid="hard-delete-option"]').should('be.visible').click();
+
+    cy.get('[data-testid="manage-button"]').should('be.visible').click();
+
+    cy.get('[data-testid="delete-button"]').should('be.visible').click();
+
+    // Check if soft delete option is present
+    cy.get('[data-testid="soft-delete-option"]').should('exist');
+
+    // Click on Permanent delete option
+    cy.get('[data-testid="soft-delete-option"]')
+      .should('contain', NEW_TEST_SUITE.name)
+      .should('be.visible')
+      .click();
```

Comment:
you are clicking on soft delete option but as per comment you said you are clicking on permanent delete option